PROTOCOL.md

```diff
@@ -424,6 +429,44 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests are sent with `REQUESTID`. `REQUESTID` is used to identify which
+action the request was intended for to protect from cases of trying to send
+multiple requests for the same action, which would otherwise effect future
+decisions. The first decision of a match has a `REQUESTID` of 1, and each
+successful decision increments the counter by 1.
```

Comment:
From what I gather based on Zarel's comment, `REQUESTID` is optional and the most recent one is filled in by the server if it was left blank. However, **if you ever need to change your decision** (using `/undo` and what not) then you need to add the `REQUESTID`. Otherwise, if the turn advanced before the `/undo` action gets processed, **your action could accidentally be used on the next request instead**.